lib/hanami/config/security.rb

```diff
@@ -50,7 +50,7 @@ def content_security_policy(value = nil)
         if value.nil?
           @content_security_policy
         else
-          @content_security_policy = value
+          @content_security_policy = value.split(';').map(&:strip).join('; ')
```

Comment:
Sorry - I probably haven't explained my point very well here. What I intended to convey was is there any validation that the user cannot input an invalid directive or option here? For example, consider the following CSP.

```
Content-Security-Policy: default *; img-src yolocopter; frame-src 'none';
```

This would cause the browser to error since `img-src: yolocopter` isn't a valid source list or keyword. This in turn renders the CSP useless as it cannot be enforced by the browser. However, from my initial comment, I'm not sure if this is something you want to support or leave it up to the developer to implement it correctly.